tests/rootfs-image-v2

```diff
@@ -29,6 +29,7 @@ else
     passive=$MENDER_ROOTFS_PART_A
     passive_num=$MENDER_ROOTFS_PART_A_NUMBER
 fi
+passive_num_hex=echo $((16#$passive_num))
```

Comment:
Isn't this missing `$(...)` around the command?

Regardless, I think it's better to keep this script compatible with `sh`, so that it can be used on minimal setups. I suggest you use this instead:

```
passive_num_hex=$(printf '%x' $passive_num)
```

Reply:
Fixed in v2

Reply:
Fixed in v2